operating/manage-user-access/configure-ocp-oauth.adoc

```diff
@@ -13,7 +13,7 @@ include::modules/configure-ocp-oauth-identity-provider.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
-* If you use a custom TLS certificate for {ocp} OAuth server, you must add the CA’s root certificate to {product-title} as a trusted root CA. Otherwise, Central cannot connect to the {ocp} OAuth server.
+* If you use a custom TLS certificate for {ocp} OAuth server, you must add the CA's root certificate to {product-title} as a trusted root CA. Otherwise, Central cannot connect to the {ocp} OAuth server.
```

Comment:
This comment is beyond the scope of this PR, but just to highlight that directories should use underscores instead of dashes (e.g. `manage_user_access`).